py/server/deephaven/parquet.py

```diff
@@ -54,6 +55,50 @@ class ParquetFileLayout(Enum):
     """
 
 
+class RowGroupInfo:
+    """ This class specifies desired RowGroup behavior """
+
+    @staticmethod
+    def single_row_group() -> _JRowGroupInfo:
+        """ All data is within a single RowGroup. This is the default RowGroupInfo implementation. """
+        return _JRowGroupInfo.singleRowGroup()
+
```

Comment:
It seems that `RowGroupInfo` should be a `JObjectWrapper`, something like:
```suggestion
class RowGroupInfo(JObjectWrapper):
    """ This class specifies desired RowGroup behavior """
    j_object_type = _JRowGroupInfo
    
    @property
    def j_object(self) -> jpy.JType:
        return self.j_row_group_info
     
     def __init__(self, j_row_group_info: jpy.JType):
         self.j_row_group_info = j_row_group_info
         
    @staticmethod
    def single_row_group() -> RowGroupInfo:
        """ All data is within a single RowGroup. This is the default RowGroupInfo implementation. """
        return RowGroupInfo(_JRowGroupInfo.singleRowGroup())

```

Also add
```
from __future__ import annotations
```
to the top of the imports.